cmd2/argcomplete_bridge.py

```diff
@@ -16,6 +16,7 @@
     except AttributeError:
         DEFAULT_COMPLETER = argcomplete.completers.FilesCompleter()
 
+    from cmd2.argparse_completer import ACTION_ARG_CHOICES, ACTION_SUPPRESS_HINT
```

Comment:
Are you happy with **argcomplete_bridge.py** depending on **argparse_completer.py**?  If so then everything looks fine to me.